legend-engine-xt-relationalStore-pure/src/main/resources/core_relational/relational/executionPlan/executionPlan.pure

```diff
@@ -44,16 +44,25 @@ Class meta::relational::mapping::TempTableColumnMetaData
    parametersForGetter      : Map<String, Any>[0..1];
 }
 
-Class meta::relational::mapping::SQLExecutionNode extends ExecutionNode
+Class meta::relational::mapping::GenericSQLExecutionNode extends ExecutionNode
 {
    sqlComment : String[0..1];
    sqlQuery : String[1];
    onConnectionCloseCommitQuery : String[0..1];
    onConnectionCloseRollbackQuery : String[0..1];
-   resultColumns : SQLResultColumn[*];
    connection: DatabaseConnection[1];
 }
 
+Class meta::relational::mapping::SQLExecutionNode extends GenericSQLExecutionNode
+{
+   resultColumns : SQLResultColumn[*];
+}
+
+Class meta::relational::mapping::SQLUpdateExecutionNode extends GenericSQLExecutionNode
```

Comment:
lets also update execution node diagram for any changes to node hierarchy

Reply:
Changed and renamed.